daml_dit_ddit/subcommand_build.py

```diff
@@ -48,6 +48,15 @@ def check_target_file(filename: str, force: bool):
         else:
             die(f'Target file already exists: {filename}')
 
+def write_pex(pex: 'ZipFile', filepath: str, arcname: 'Optional[str]' = None, filebytes: 'Optional[bytes]' = None):
+    filename = filepath.split("/")[-1]
+    if filename in pex.namelist():
+        LOG.warn(f'File {filename} exists in archive -- skipping.')
+    else:
+        if filebytes:
+            pex.writestr(filepath, filebytes)
+        else:
+            pex.write(filepath, arcname=arcname)
```

Comment:
This needs to be two separate functions for the two purposes it serves. The conditional (lines 52-53) can probably be factored out into a separate function called by the two others.)